storage/config.go

```diff
@@ -39,11 +43,15 @@ type Configuration struct {
 	Database string `yaml:"database"`
 	// Endpoint for scraping prometheus metrics e.g. localhost:9090.
 	MetricsEndpoint string `yaml:"metrics_endpoint"`
-	// Table with spans. Default "jaeger_spans_local".
+	// Whether to use SQL scripts supporting replication and sharding. Default false.
+	Replication bool `yaml:"replication"`
+	// Any table name in config is used for global distributed table.
+	// For non-distributed/local tables '{config table name}_local' is used.
+	// Table with spans. Default "jaeger_spans".
 	SpansTable string `yaml:"spans_table"`
-	// Span index table. Default "jaeger_index_local".
+	// Span index table. Default "jaeger_index".
```

Comment:
revert this keep `_local`  as the default.

Reply:
I wonder what we should do with custom database names. Should we require them to end with '_local'? In this case we can simply get global tablenames.

Reply:
It's user's responsibility and decision to use the right names.

Reply:
@pavolloffay How to get global table then? Should we simply delete last 6 symbols from table name?